xml/security_ldap.xml

```diff
@@ -389,290 +387,57 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
-    <para>
-     In the window <guimenu>LDAP and Kerberos Client</guimenu>, click
-     <guimenu>Change Settings</guimenu>.
-    </para>
-    <para>
-     Make sure that the tab <guimenu>Use a Directory as Identity Provider
-     (LDAP)</guimenu> is chosen.
-    </para>
-    <informalfigure>
-     <mediaobject>
-      <imageobject role="html">
-       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
-      </imageobject>
-      <imageobject role="fo">
-       <imagedata fileref="yast2_auth_client_ldap.png" width="65%" format="PNG"/>
-      </imageobject>
-     </mediaobject>
-    </informalfigure>
-   </step>
-   <step>
-    <para>
-     Specify one or more LDAP server URLs, host names, or IP addresses under
-     <guimenu>Enter LDAP server locations</guimenu>. When specifying multiple
-     addresses, separate them with spaces.
-    </para>
-   </step>
-   <step>
-    <para>
-     Specify the appropriate LDAP distinguished name (DN) under
-     <guimenu>DN of Search Base</guimenu>. For example, a valid entry could
-     be <literal>dc=example,dc=com</literal>.
-    </para>
-   </step>
-   <step>
-    <para>
-     If your LDAP server supports TLS encryption, choose the appropriate
-     security option under <guimenu>Secure LDAP Connection</guimenu>.
-    </para>
-    <para>
-     To first ask the server whether it supports TLS encryption and be able
-     to downgrade to an unencrypted connection if it does not, use
-     <guimenu>Secure Communication via StartTLS</guimenu>.
-    </para>
-   </step>
-   <step>
-    <para>
-     Activate other options as necessary:
-    </para>
-    <itemizedlist>
+  <variablelist>
+   <varlistentry>
+    <term><command>dsctl</command></term>
+    <listitem>
+     <para>
+      Manages a local instance and requires &rootuser; permissions. Used for
+      starting, stopping, backing up the database and more.</para>
+    </listitem>
+   </varlistentry>
+    <varlistentry>
+     <term><command>dsconfig</command></term>
      <listitem>
       <para>
-       You can <guimenu>Allow users to authenticate via LDAP</guimenu> and
-       <guimenu>Automatically Create Home Directories</guimenu> on the local
-       computer for them.
+       Manages a remote or local instance configuration. Requires
+       <literal>cn=Directory Manager</literal>. The primary tool used for
+       administration and configuration of the server.
       </para>
      </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term><command>dsidm</command></term>
      <listitem>
       <para>
-       Use <guimenu>Cache LDAP Entries For Faster Response</guimenu> to cache
-       LDAP entries locally. However, this bears the danger that entries can
-       be slightly out of date.
+       Manages content inside of a back-end. The permissions of this tool are
+       granted by access controls. <remark>taroth 2019-02-25: not sure what the
+       following  sentence is referring to? "and can even be used for some
```

Comment:
So the dsidm tool manages things like users, groups, passwords etc. So some "self-service" actions would include a person being able to reset their own password or change details of their own account with access-controls permitting.

Reply:
Added.